### DISCUSSION TITLE
[solved] check for existing drive in windows

### DISCUSSION THREAD

**Discussion by airathere** [ConSol-Monitoring/snclient#167] [General]
Hi all,

i would like to know if there is a possibility to check for a drive in windows, e.g. drive E:
As far as i can see there is a possibility for checking a drivesize or a file, but i have the problem, that the to be checked drive is somtimes empty (and has to be!) and a check against size or file is needless.

Thanks for any answer on this!
Joe

**Comment by sni** [comment]
There is a `check_mount`. Maybe that one fits better. But you could also adjust the thresholds of `check_drivesize`.

**Reply by airathere** [reply]
Ah OK, i didn't expected, that check_mount is possible, because https://github.com/ConSol-Monitoring/snclient?tab=readme-ov-file#check-plugin-status shows no "X" (support) for check_mount under Windows?!

I tried it now and it works 👍

OMD[me@omd]:~$ check_nrpe -H 192.168.178.10 -S TLSv1.2+ -p 5666 -c check_mount -a mount=E:
CRITICAL - mount E: not mounted

So, all is fine - thank you very much for your help!

**Reply by sni** [reply]
> Ah OK, i didn't expected, that check_mount is possible, because https://github.com/ConSol-Monitoring/snclient?tab=readme-ov-file#check-plugin-status shows no "X" (support) for check_mount under Windows?!

Thanks, missed that somehow. I added some more X
